teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(id, name, age, specialization, parents_permission = 'Unknown')
+    @specialization = specialization
+    super(id, name, age, parents_permission)
+  end
+
+  def can_use_services?
+    puts true
```

Comment:
- [ ] Please as mentioned by the previous reviewer, this method should return `true` and not print the value `true`, kindly fix that 🙃 
eg: 
```ruby
def can_use_services?
    true
end
```